libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/TableList.tsx

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  FocusZone,
+  DetailsList,
+  DetailsHeader,
+  IDetailsHeaderProps,
+  IRenderFunction,
+  IGroup,
+  IColumn,
+  Image,
+  Text,
+  Stack
+} from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { IDatasetSummary } from "../Interfaces/IExplanationDashboardProps";
+
+export interface ITableListProps {
+  data: IDatasetSummary;
+  imageDim: number;
+  pageSize: number;
+  selectItem: (item: IListItem) => void;
+}
+
+export interface ITableListState {
+  items: IListItem[];
+  groups: IGroup[];
+  columns: IColumn[];
+}
+
+export interface IListItem {
+  title: string;
+  subtitle?: string;
+  image: string;
+  trueY?: number;
+  predictedY?: number;
+  index?: number;
+  other?: number;
+}
+
+export class TableList extends React.Component<
+  ITableListProps,
+  ITableListState
+> {
+  public constructor(props: ITableListProps) {
+    super(props);
+
+    this.state = {
+      columns: [],
+      groups: [],
+      items: []
+    };
+  }
+
+  public componentDidMount(): void {
+    const items: IListItem[] = [];
+
+    this.props.data.images.forEach((image, index) => {
+      items.push({
+        image,
+        index: index + 1,
+        other: 0,
+        predictedY: 1,
+        subtitle: "subtitle",
+        title: `label ${(index + 1).toString()}`,
+        trueY: 0
+      });
+    });
+
+    const groups: IGroup[] = [
+      {
+        count: this.props.pageSize,
+        key: "success",
+        level: 0,
+        name: localization.InterpretVision.Dashboard.titleBarSuccess,
+        startIndex: 0
+      },
+      {
+        count: this.props.pageSize,
+        key: "error",
+        level: 0,
+        name: localization.InterpretVision.Dashboard.titleBarError,
+        startIndex: this.props.pageSize
+      }
+    ];
+
+    const columns: IColumn[] = [
+      {
+        fieldName: "title",
+        isResizable: true,
+        key: "title",
+        maxWidth: 400,
+        minWidth: 200,
+        name: localization.InterpretVision.Dashboard.columnOne
+      },
+      {
+        fieldName: "index",
+        isResizable: true,
+        key: "index",
+        maxWidth: 400,
+        minWidth: 200,
+        name: localization.InterpretVision.Dashboard.columnTwo
+      },
+      {
+        fieldName: "trueY",
+        isResizable: true,
+        key: "truey",
+        maxWidth: 400,
+        minWidth: 200,
+        name: localization.InterpretVision.Dashboard.columnThree
+      },
+      {
+        fieldName: "predictedY",
```

Comment:
nit: it might be cleaner to build the fields in a loop to reduce code duplication